projects/plugins/wpcomsh/README.md

```diff
@@ -11,13 +11,19 @@ With the help of this mu-plugin, an Atomic site is transformed into a WP.com sit
 ### Quick Start
 
 ```
+# Clone the Monorepo somewhere locally
+$ git clone git@github.com:Automattic/jetpack.git
+$ cd jetpack
+$ pnpm install
+$ pnpm jetpack build --deps projects/wpcomsh
+
+# Specify your development server and path to sync the code to your wp.org install 
+$ pnpm jetpack rsync projects/wpcomsh USER@HOST:/path/to/wordpress/wp-content/mu-plugins 
+
 # From the root of a wp.org install
 $ mkdir -p wp-content/mu-plugins
 $ cd wp-content/mu-plugins
-$ git clone git@github.com:Automattic/wpcomsh.git
-$ cd wpcomsh
-$ composer install # installs the composer dependencies
-$ cd ..
+$ cd jetpack/projects/plugins/wpcomsh
```

Comment:
This won't work now. In the current context where we've rsynced the plugin, you'd just want
```suggestion
$ cd wpcomsh
```
(or simpler to combine this with line 25 as `cd wp-content/mu-plugins/wpcomsh`, but GH won't let me suggest that)

Reply:
Oh, you're right. I don't think we need it at all, because the next command is the symlinking, and it should be run in the context of `mu-plugins`, not `wpcomsh`.